swagger/v2/api_spec.json

```diff
@@ -66,24 +66,10 @@
         ],
         "responses": {
           "200": {
-            "description": "A list of NPQ applications",
```

Comment:
could you add those back please, as we no longer can access the [npq application](https://cpd-ecf-review-3882-web.test.teacherservices.cloud/api-reference/reference-v2.html#api-v2-npq-applications-get) response?